ddtrace/appsec/_handlers.py

```diff
@@ -20,6 +20,19 @@
 _BODY_METHODS = {"POST", "PUT", "DELETE", "PATCH"}
 
 
+def _get_content_length(environ):
+    content_length = environ.get("CONTENT_LENGTH")
+    transer_encoding = environ.get("HTTP_TRANSFER_ENCODING")
+
+    if transer_encoding == "chunked" or content_length is None:
```

Comment:
```suggestion
    transfer_encoding = environ.get("HTTP_TRANSFER_ENCODING")

    if transfer_encoding == "chunked" or content_length is None:
```

I cannot spell apparently.